packages/mongo-nav/src/mongo-select/OrgSelect.tsx

```diff
@@ -228,7 +238,16 @@ function OrgSelect({
           </span>
 
           {!isOnPrem && (
-            <span className={productStyle}>{formattedPlanTypes[planType]}</span>
+            <span className={displayFlex}>
```

Comment:
Let's use a div since we're manually setting the display property anyways.

Reply:
👍